### ISSUE TITLE
Formatter adds spaces to get_node $/root paths

### ISSUE THREAD

**Issue by m-radzikowski** [godotengine/godot-vscode-plugin#710]
### Godot version

v4.3.stable.official [77dcf97d8]

### VS Code version

1.93.0

### Godot Tools VS Code extension version

2.1.0

### System information

Windows 11

### Issue description

For get_node `$` syntax, if path starts with `/`, the formatter adds spaces around all `/`.

This:
```
var main = $/root/Main
```

is changed to:

```
var main = $ / root / Main
```

The syntax still works, but that's not a proper formatting.

A workaround is to wrap path in quotes:

```
var main = $"/root/Main"
```

It looks like the `$/` syntax is not recognized, because the code highlighting of `$` is different for `$node` and `$/root`.

### Steps to reproduce

Write:

```
var main = $/root/Main
```

and trigger code formatting.

**Comment by DaelonSuzuka** [comment]
Wow, I don't know how I missed that this was valid syntax! Thanks for the report

**Comment by DaelonSuzuka** [comment]
This should be fixed in the next release.

![Code_Tn5BPd7TN4](https://github.com/user-attachments/assets/27802db9-09bf-4c68-9cbb-7a9673b77028)

**Comment by m-radzikowski** [comment]
Thank you for such a quick reaction!

**Comment by DaelonSuzuka** [comment]
@m-radzikowski  The fix for this issue was just published in `v2.2.0`. Please let me know if you're still experiencing this after you update.
